misc/taskmetadata-validator/hello.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
I don't feel strongly about this. But, you're probably better off using a base image like `alpine` and using one of the commands available there as a proxy instead of doing something like this. Means less code and less chance for bugs.

Reply:
yeah, I thought the same. I'll modify that.